docs/usage.md

```diff
@@ -207,105 +255,326 @@ which are optional:
-      name: slurm
-      version: 1.1.1
-      branch: master
+      version: "{{cpe.version}}"
+      branch: integration-{{cpe.version}}
+
 images:
-- name: cray-shasta-compute-sles15sp3.x86_64-2.2.35
-  ims:
-    is_recipe: true
-    name: cray-shasta-compute-sles15sp3.x86_64-2.2.35
-  configuration: cos-config
+- name: "{{base.name}}"
+  ref_name: base_cos_image
+  base:
+    product:
+      name: cos
+      type: recipe
+      version: "{{cos.version}}"
+
+- name: compute-{{base.name}}
+  ref_name: compute_image
+  base:
+    image_ref: base_cos_image
+  configuration: compute-{{recipe.version}}
   configuration_group_names:
   - Compute
+
 session_templates:
-- name: cray-shasta-compute-sles15sp3.x86_64-2.2.35
-  image: cray-shasta-compute-sles15sp3.x86_64-2.2.35
-  configuration: cos-config
+- name: compute-{{recipe.version}}
+  image:
+    image_ref: compute_image
+  configuration: compute-{{recipe.version}}
   bos_parameters:
     boot_sets:
       compute:
         kernel_parameters: ip=dhcp quiet spire_join_token=${SPIRE_JOIN_TOKEN}
         node_roles_groups:
         - Compute
+        rootfs_provider_passthrough: "dvs:api-gw-service-nmn.local:300:hsn0,nmn0:0"
 ```
 
-#### Creating a pre-populated example bootprep input file
+#### Accessing default bootprep input files
+
+Default bootprep input files are delivered by the HPC CSM Software Recipe
+product. These files can be accessed by cloning the `hpc-csm-software-recipe`
+repository.
 
-It is possible to create an example bootprep input file using values from the
-system's product catalog using the `sat bootprep generate-example` command.
+Follow steps 1-7 of the procedure in [Viewing HPC CSM Software Recipe
+variables](#viewing-hpc-csm-software-recipe-variables).
+
+Access the files in the `bootprep` directory of that repository:
 
 ```screen
-ncn-m001# sat bootprep generate-example
-INFO: Using latest version (2.3.24-20220113160653) of product cos
-INFO: Using latest version (21.11.4) of product cpe
-INFO: Using latest version (1.0.7) of product slurm
-INFO: Using latest version (1.1.24) of product analytics
-INFO: Using latest version (2.1.5) of product uan
-INFO: Using latest version (21.11.4) of product cpe
-INFO: Using latest version (1.0.7) of product slurm
-INFO: Using latest version (1.1.24) of product analytics
-INFO: Using latest version (2.3.24-20220113160653) of product cos
-INFO: Using latest version (2.1.5) of product uan
-INFO: Wrote example bootprep input file to ./example-bootprep-input.yaml.
+ncn-m001# ls bootprep/
 ```
 
-This file should be reviewed and edited to match the desired parameters of the
-configurations, images, and session templates.
+#### Generating an example bootprep input file
+
+Note: the `sat bootprep generate-example` command has not been updated for
+recent bootprep schema changes, and it is recommended to use the default
+bootprep input files described in [Accessing default bootprep input
+files](#accessing-default-bootprep-input-files).
+
```

Comment:
I would suggest removing line 575 so all the information in this section is one paragraph.